flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/config/FlinkConfigManager.java

```diff
@@ -134,6 +135,11 @@ public Configuration getDeployConfig(ObjectMeta objectMeta, FlinkDeploymentSpec
     public Configuration getObserveConfig(FlinkDeployment deployment) {
         var deployedSpec = ReconciliationUtils.getDeployedSpec(deployment);
         if (deployedSpec == null) {
+            try {
+                LOG.debug("deployment: {}", objectMapper.writeValueAsString(deployment));
+            } catch (JsonProcessingException e) {
+                LOG.debug("deployment: {}", deployment);
+            }
```

Comment:
I think this debug log does not add any value here. Could we please remove it?

Reply:
Ok